integration-tests/deployment/ccip/view/types/contract_state.go

```diff
@@ -0,0 +1,33 @@
+package types
```

Comment:
The type direction is to avoid import cycle, We have the Contract struct getting imported into v1_2 ,v1_5 etc from parent view and then Chain struct has elements from v1_2, v1_5 in the parent view package

Reply:
I see - I think it could be on the same hierarchical level though just to avoid the nesting